debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (23.3.1-0ubuntu2) UNRELEASED; urgency=medium
+
+  * New upstream bug fix release based on 23.3.1.
+  * Install gnupg and software-properties-common
+    in cc_apt_configure if included in base image.
```

Comment:
We should provide the proper indent of scope prefix ` -` for the changes included in the new upstream bug fix. Let's also correct the message as we only install if the pkgs are absent and cloud-config requires those cmds

```suggestion
    + Install gnupg and software-properties-common
      if absent in base image but required by apt cloud-config
```